src/js/BookReader.js

```diff
@@ -360,6 +360,10 @@ BookReader.prototype.init = function() {
     }
   }
 
+  if (this.options.startFullscreen) {
```

Comment:
Let's add this option to `options.js` with a default of `false`, and a docstring :) That way others can know it exists, since that's our current source of truth (or at least the closest thing we have to a source of truth :D )